app/database/query.service.js

```diff
@@ -0,0 +1,10 @@
+import DbConnectionService from './db-connection.service';
+export default class QueryService {
+    executeQueryString(sqlString) {
+        return DbConnectionService.getPool().query(sqlString);
+    }
```

Comment:
I would consolidate the two functions into one, called 'executeQuery'.
The resulting function could have a signature, as follows:
executeQuery(sqlStatement, params = null);